ff2mpv.py

```diff
@@ -47,6 +96,22 @@ def get_message():
     return json.loads(message)
 
 
+def get_whitelist():
+    path = get_config_path('whitelist.txt')
```

Comment:
Oh, I see how you're doing this. Instead of having it be `whitelist.txt`, let's just make `~/.config/ff2mpv` a plain text file. Inflexibility is a virtue here.

Reply:
Hm but i do think a single whitelist.txt file is way easier to understand for users. I know atm there is no need for an additional config, but I think this is way cleaner, than just having a file with a generic name without a folder corresponding to the app.

Reply:
I think flexibility is very important when it comes to config dirs, especially since you want to maintain compatibility with three different platforms. In fact I think a sensible fallback for Windows should also be added, but I have no clue, where Windows users prefer their config files....

Reply:
Handing over the filename is just so you don't have to add another `os.path.join()` on every/any method you call it. Depending on the perspective this could also be considered "simpler"...

Reply:
Also if you leave it blank it just falls back to giving you the config directory name/path anyway.

Reply:
Okay, you've sold me on it being a separate file. `<config-root>/whitelist` seems reasonable to me, going with the convention of not generally including suffixes for configuration files.

Reply:
(Also, for context: ff2mpv's support for Windows should be considered second-tier at best. It's perfectly okay, in my book, for us to just use the normal `Path.home()` resolution and drop a dot-directory there.)

Reply:
Ok I renamed it to `whitelist`.